apps/README.md

```diff
@@ -0,0 +1,19 @@
+# Stork Apps
+
+A suite of tools to interact with Stork's services and on-chain contracts.
+
+## pusher
```

Comment:
nit - do we want to use `Pusher` and `Publisher Agent` instead of the snake case versions for the readme titles? Fine with this if it's a convention or something